requirements-dev.txt

```diff
@@ -32,7 +32,7 @@ pytest
 scipy
 selenium<4.27.0
 setuptools_scm
-sphinx
+sphinx==7.3.7
```

Comment:
What are the errors with the newer versions?

Reply:
`exception: module 'sphinx.util' has no attribute 'console'`

Reply:
I'll check that next week. Let's merge this one for now. Thanks!